src/domain/occurrence/utils.ts

```diff
@@ -3,13 +3,24 @@ import { isPast, subDays } from 'date-fns';
 import {
   EventFieldsFragment,
   OccurrenceFieldsFragment,
+  OccurrenceSeatType,
 } from '../../generated/graphql';
+import assertUnreachable from '../../utils/assertUnreachable';
 
 export const hasOccurrenceSpace = (
   occurrence: OccurrenceFieldsFragment
 ): boolean => {
-  const minGroupSize = occurrence?.minGroupSize || 0;
-  return minGroupSize < occurrence.amountOfSeats - (occurrence.seatsTaken || 0);
+  switch (occurrence.seatType) {
+    case OccurrenceSeatType.ChildrenCount:
+      const minGroupSize = occurrence?.minGroupSize || 0;
+      return (
+        minGroupSize < occurrence.amountOfSeats - (occurrence.seatsTaken || 0)
```

Comment:
Small thing but it seems that amount of seats left is calculcated in few places, it could be in a util function:
`OccurenceUtils.getAmountOfReservedSeats: (occurence) => occurrence.amountOfSeats - (occurrence.seatsTaken || 0)`

Reply:
Hmm would it be `OccurenceUtils.getAmountOfSeatsLeft` instead?

Reply:
Fixed

Reply:
That's good name as well